.github/workflows/builder_gradle_slsa3.yml

```diff
@@ -50,6 +50,13 @@ on:
         description: "The sha256 of the provenance attestation uploaded to the workflow run."
         value: "${{ jobs.slsa-run.outputs.attestations-download-sha256 }}"
 
+      target-download-name:
```

Comment:
is there a reason the gradle builder uses 'build' rather than `target`(Maven) internally for naming the output build directory? Is this a naming convention followed by gradle users? If build is more common, shall we rename `target-` to `build-` in the gradle builder?

/cc @loosebazooka

Reply:
That is exactly the reason. Gradle creates a `build` directory and Maven creates a `target` directory. Will change the naming convention in the builder to reflect that.

Reply:
Should have fixed all names.